src/scos_tekrsa/configs/actions-500-600/CBRS_DataProduct_00dB_Attenuation.yml

```diff
@@ -2,29 +2,26 @@ nasctn_sea_data_product:
   name: CBRS_DataProduct_00dB_Attenuation
   rf_path: antenna
 # IIR filter settings
-  # iir_apply: True
   iir_gpass_dB: 0.1  # Max passband ripple below unity gain
   iir_gstop_dB: 40  # Minimum stopband attenuation
   iir_pb_edge_Hz: 5e6  # Passband edge frequency
   iir_sb_edge_Hz: 5.008e6  # Stopband edge frequency
 # Mean/Max FFT settings
-  # fft_size: 875
   nffts: 64e3
-  # fft_window_type: flattop # See scipy.signal.get_window for supported input
 # PFP frame
   pfp_frame_period_ms: 10
 # APD downsampling settings
-  apd_bin_size_dB: 0.5  # Set to 0 or negative for no downsampling
+  apd_bin_size_dB: 1.0  # Set to 0 or negative for no downsampling
+  apd_max_bin_dBm: -30
+  apd_min_bin_dBm: -180
 # Time domain power statistics settings
   td_bin_size_ms: 10
-# Round all power results to X decimal places
-  # round_to_places: 2
 # Sigan Settings
   preamp_enable: True
   reference_level: -25
   attenuation: 0
   sample_rate: 14e6
-# Acquisition settings (3550-3700 MHz in 10 MHz steps, each 4s long)
+# Acquisition settings (3540-3700 MHz in 10 MHz steps, each 4s long)
```

Comment:
Isn't this still 3550?

Reply:
Yep- the config is right but this comment was incorrect. Fixed in [3d48f6f](https://github.com/NTIA/scos-tekrsa/pull/39/commits/3d48f6fa4a387b17583ea9afcebc49a1cf7d0038)